src/vs/workbench/parts/search/test/common/queryBuilder.test.ts

```diff
@@ -473,6 +473,14 @@ suite('QueryBuilder', () => {
 						}]
 					}
 				],
+				[
+					'../',
+					<ISearchPathsResult>{
+						searchPaths: [{
+							searchPath: getUri('foo/')
```

Comment:
Should it be an absolute path?

Reply:
I think this is correct as is, ultimately the _formatted properties of the uris are compared, which are both file:///foo. Follows same convention as other tests here